src/artemis/external_interaction/callbacks/fgs/nexus_callback.py

```diff
@@ -40,29 +43,31 @@ def start(self, doc: dict):
             )
             json_params = doc.get("hyperion_internal_parameters")
```

Comment:
Could: I still think it would be cleaner to just do this on any doc that contains `hyperion_internal_parameters`, regardless of name.